wallet/service.go

```diff
@@ -137,33 +163,19 @@ func NewServiceWith(log *zap.Logger, cfg *Config, rootPath string, h WalletHandl
 	s.PUT("/api/v1/keys/:keyid/taint", ExtractToken(s.TaintKey))
 	s.PUT("/api/v1/keys/:keyid/metadata", ExtractToken(s.UpdateMeta))
 	s.POST("/api/v1/sign", ExtractToken(s.SignAny))
+	s.POST("/api/v1/command", ExtractToken(s.SignTxV2))
+	s.POST("/api/v1/command/sync", ExtractToken(s.SignTxSyncV2))
```

Comment:
Could we default /command to be sync, remove the /command/sync and add a /command/async here? I did the same for the tx v1 recently, as it helps with error returns